.circleci/config.yml

```diff
@@ -35,7 +35,7 @@ jobs:
       - checkout
       - restore_cache:
           key: sbtcache
-      - run: sbt ++2.13.3! core/test http4s/compile akkaHttp/compile finch/compile play/compile examples/compile catsInterop/compile monixInterop/compile tapirInterop/test clientJVM/test federation/test
+      - run: sbt ++2.13.3! core/test http4s/compile akkaHttp/compile finch/compile play/compile examples/compile catsInterop/compile monixInterop/compile tapirInterop/test clientJVM/test federation/test codegenSbt/scripted
```

Comment:
I think the sbt plugin only compiles with 2.12

Reply:
Thanks!
I've changed it. I hope the CI will be success.